content/_data/changelogs/weekly.yml

```diff
@@ -6190,6 +6190,95 @@
   # pull: 4474 (PR title: Enforce singleton pattern in CaseInsensitiveComparator)
   # pull: 4481 (PR title: Release Drafter: Do not use `regression-fix` label for the changelog, use the global template)
 
+- version: '2.221'
+  date: 2020-02-15
+  changes:
+  - type: major rfe
+    category: major rfe
+    pull: 4368
+    authors:
+    - daniel-beck
+    message: |-
+      Add globally configured build discarders that delete old builds not marked as "keep forever" even if there is no, or a less aggressive, per-project build discarder configured, executed periodically and after a build finishes.
+  - type: major bug
+    category: major bug
+    pull: 4368
+    authors:
+    - daniel-beck
+    message: |-
+      Jenkins will by default execute the configured per-project build discarder periodically even if no build is currently finishing. <strong>This may delete old builds of projects that got a more aggressive build discarder configuration since the last build was run.</strong>
```

Comment:
Should it be in the changelog summary instead?

Reply:
Can you explain further?  The pull request 4368 is intentionally listed 3 times, once for a major rfe, once for a major bug, and once for a developer rfe.

Are you suggesting that the `<strong> ` text should be earlier in the message or as a summary of the entire release or in some other location that I haven't understood?

Reply:
Summary of the entire release. This one looks like a pretty serious warning, so IMHO it worth it. Not blocking by it